api/src/main/scala/org/ensime/api/common.scala

```diff
@@ -23,9 +23,10 @@ object DeclaredAs {
   case object Object extends DeclaredAs('object)
   case object Class extends DeclaredAs('class)
   case object Field extends DeclaredAs('field)
+  case object TypeAlias extends DeclaredAs('typeAlias)
```

Comment:
Aha, you're right. I don't think this is critical either TBH. It just seemed strange that the `DeclaredAs` value would be `Nil`.

What do you say? Shall we drop it?